### PR TITLE
Redisson param beautify

### PR DESCRIPTION
**Pull request by w-a-n-g-s-h-u-n**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

File: apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java [hunk 1]
```diff
@@ -88,7 +87,14 @@ private void addCommandData(StringBuilder dbStatement, CommandData commandData)
         if (RedissonPluginConfig.Plugin.Redisson.TRACE_REDIS_PARAMETERS && commandData.getParams() != null) {
             for (Object param : commandData.getParams()) {
                 dbStatement.append(DELIMITER_SPACE);
-                String paramStr = param instanceof ByteBuf ? QUESTION_MARK : String.valueOf(param.toString());
+                String paramStr;
+                if (param instanceof ByteBuf) {
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java:91]: Could you provide what is a bytebuf?

**Comment by wu-sheng** [comment]
Also, the `changes.md` in the root should be updated accordingly.

**Comment by wu-sheng** [comment]
You should fix CI.

File: apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java (hunk 1 above)
Comment (w-a-n-g-s-h-u-n) [reply apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java:91]: > Could you provide what is a bytebuf?

I get results like "EXISTS [B@1ce73d9a",But I'm not sure it's always like that

`if (param instanceof ByteBuf)` may always be `false`

**Comment by w-a-n-g-s-h-u-n** [comment]
> You should fix CI.

Done

> You should fix CI.

File: apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java (hunk 1 above)
Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java:91]: How does the latest look like? Could you post some examples?

**Comment by wu-sheng** [comment]
The plugin test verified your changes, and fails, https://github.com/apache/skywalking-java/actions/runs/3217108062/jobs/5260283393

**Comment by w-a-n-g-s-h-u-n** [comment]
> The plugin test verified your changes, and fails, https://github.com/apache/skywalking-java/actions/runs/3217108062/jobs/5260283393

I gave up on modifying the ByteBuf case. In fact, I didn't figure out how the test case works

File: apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java [hunk 2]
```diff
@@ -88,7 +88,14 @@ private void addCommandData(StringBuilder dbStatement, CommandData commandData)
...
+                    paramStr = QUESTION_MARK;
+                } else if (param instanceof byte[]) {
+                    paramStr = new String((byte[]) param);
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java:95]: Are you sure the byte[] could be serialized as a String?

**Comment by wu-sheng** [comment]
@w-a-n-g-s-h-u-n Are you going to continue on this PR, we are waiting for your explanaion.

**Comment by wu-sheng** [comment]
This PR has no update in half month. I have to close this.
If you want to continue on this, let's know.

**Pull request closed without merging**